travel-provider/TSP-30-RankTravelOptions/TSP-30-RankTravelOptions-RequestSchema.json

```diff
@@ -0,0 +1,83 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "title": "TSP-30-RankTravelOptionsRequest Request Schema",
+  "version": "0.0.1",
+  "description": "Schema for ranking travel options request sent by CWT",
+  "properties": {
+    "returnId": {
+      "type": "integer",
+      "description": "Unique Id for this booking - generated by RMP."
```

Comment:
Clarify description: "Unique Id for this booking - take from the original booking request sent from RMP to TSP."